ignite/handlers/checkpoint.py

```diff
@@ -30,7 +30,7 @@ def remove(self, filename: str) -> None:
 class Checkpoint:
     """Checkpoint handler can be used to periodically save and load objects which have attribute
     `state_dict`/`load_state_dict`. This class can use specific save handlers to store on the disk or a cloud
-    storage, etc. The Checkpoint handler also handles automatically moving devices on TPU to CPU before writing
+    storage, etc. The Checkpoint handler also handles automatically moving data on TPU to CPU before writing
```

Comment:
Actually, this is not done by Checkpoint but DiskSaver ?

Reply:
Technically yes. My thought was that Since the `Checkpoint` handler defers to the `DiskSaver` and since I imagine most people work through `*Checkpoint`, including that info here would make most sense. I am happy to move it as needed.